cmd/root.go

```diff
@@ -73,3 +80,29 @@ func getContainer() *core.Container {
 	}
 
 }
+
+func setStorage() error {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	configPath := path.Join(home, ".config", "containers", "storage.conf")
```

Comment:
that path does not even exist on my setup, I think we need to make it before writing the conf:

```go
// storage config doesn't exist
err = os.MkdirAll(path.Join(home, ".config", "containers"), 0755)
if err != nil {
	return err
}
```

Reply:
great catch! thanks.